README.md

```diff
@@ -341,22 +346,30 @@ Testing Time: 3.96s
   Unsupported Tests  : 5
 ```
 
-In the above example we see that twenty-three tests succeeded and five were unsupported.
+In the above example we see that 23 tests succeeded and 5 were unsupported.
 
 ### Result Code Values
 
-Our tests use the standard [lit result codes][] and the addition of a non-standard result code `SKIP`. For our tests
-one need mostly only concern themselves with the `PASS`, `XFAIL`, `XPASS`, `FAIL`, and `UNSUPPORTED` result codes.
+Our tests use the standard [lit result codes][], and a non-standard result code: `SKIP`. For our tests, only the
+`PASS`, `XFAIL`, `XPASS`, `FAIL`, and `UNSUPPORTED` standard result codes are relevant.
 
-The `PASS` and `FAIL` result codes are self explanatory. We want our tests to `PASS` and not `FAIL`.
-The `XPASS` and `XFAIL` are less obvious. `XFAIL` is actually a successful result and indicates that we expected the
+The `PASS` and `FAIL` result codes are self-explanatory. We want our tests to `PASS` and not `FAIL`.
 
-test to fail and it did. `XPASS` is a failure result and indicates that we expected a test to fail but it passed.
-Typically this means that the `expected_results.txt` file for the testsuite needs to be modified and a `FAIL` entry
-needs to be removed.
+The `XPASS` and `XFAIL` result codes are less obvious. `XPASS` is actually a failure result and indicates that we
+expected a test to fail but it passed. `XFAIL` is a successful result and indicates that we expected the test to fail
+and it did. Typically an `XPASS` result means that the `expected_results.txt` file for the testsuite needs to be
+modified. If the `XPASS` result is a test legitimately passing, the usual course of action would be to remove a `FAIL`
+entry from the `expected_results.txt`. However, some tests from `libcxx` mark themselves as `XFAIL` (meaning they
+expect to fail) for features they have added tests for but have yet to implement in `libcxx`. If the STL implements
+those features first the tests will begin passing unexpectedly for us and return `XPASS` results. In order to resolve
+this it is necessary to add a `PASS` entry to the `expected_results.txt` of the testsuite in question.
 
 The `UNSUPPORTED` result code means that the requirements for a test are not met and so it will not be run. Currently
-all tests which use the `/BE` or `clr:pure` flags are unsupported.
+all tests which use the `/BE` or `/clr:pure` options are unsupported.
+
+The `SKIP` result code indicates that a given test was explicitly skipped by adding a `SKIP` entry to the
+`expected_results.txt`. A test may be skipped for a number or reasons, which include, but are not limited to: it being
+an incorrect test; it taking a very long time to run; it failing or passing for the incorrect reason.

```

Comment:
Fix "number or reasons" typo and change semicolon-delimited list to normal bullet points:
```suggestion
`expected_results.txt`. A test may be skipped for a number of reasons, which include, but are not limited to:
* being an incorrect test
* taking a very long time to run
* failing or passing for the incorrect reason
```